schedule/sles4sap_offline_gnome.yaml

```diff
@@ -0,0 +1,39 @@
+---
+name:           sles4sap_offline_gnome
+description:    >
+  Graphical installation tests for SLES for SAP Applications without
+  SCC registration. Boot into Gnome.
+  Test SLES for SAP specific applications.
+vars:
+  APPTESTS: sles4sap/patterns,sles4sap/sapconf,sles4sap/saptune
+  DM_NEEDS_USERNAME: 1
+  HDDSIZEGB: 60
+  ROOTONLY: 1
+  SCC_REGISTER: never
+  SCC_URL:
+  SHUTDOWN_NEEDS_AUTH: 0
+  SLES4SAP_MODE: sles4sap
+  SLE_PRODUCT: sles4sap
+  SYSTEM_ROLE: default
+  WORKAROUND_MODULES: base,desktop,sapapp,serverapp,ha
+schedule:
+  - installation/bootloader
+  - installation/welcome
+  - installation/accept_license
+  - installation/registration
+  - installation/addon_products_sle
+  - installation/system_role
```

Comment:
you're missing here: `sles4sap_product_installation_mode`, please add it as well.

Reply:
Done. although i couldnt find the job to verify it